modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for Cluster Observability Operator
+
+To be able to discover the sample `ServiceMonitor` resource you created, you must create and configure a `MonitoringStack` object.
+
+This procedure shows how to create a `MonitoringStack` object to discover the sample `ServiceMonitor` resource for a sample service named `prometheus-coo-example-app` in the `ns1-coo` namespace.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.
+* You have installed the Cluster Observability Operator.
+* You have deployed the `prometheus-coo-example-app` sample service in the `ns1-coo` project.
+* You have configured a custom resource to specify how the service is to be monitored.
+
+
+.Procedure
+
+. Create a YAML file for the `MonitoringStack` object configuration. For this example, name the file `example-coo-monitoring-stack.yaml`.
+
+. Add the following `MonitoringStack` object configuration details:
++
+[source,yaml]
+----
+apiVersion: monitoring.rhobs/v1alpha1
+kind: MonitoringStack
+metadata:
+  name: example-coo-monitoring-stack
+  namespace: ns1-coo
+spec:
+  logLevel: debug
+  retention: 1d
+  resourceSelector:
+    matchLabels:
+      k8s-app: prometheus-coo-example-monitor
+----
+
+. Apply the configuration to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f example-coo-monitoring-stack.yaml
+----
+
+. Verify that the `MonitoringStack` object is available:
```

Comment:
```suggestion
. Verify that the `MonitoringStack` object is available, by running the following command and inspecting the output:
```

Reply:
Added everything but the comma, which I don't think is needed.